clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -180,21 +182,24 @@ func (e *etcdMetricsMeasurement) stopAndSummarize(host, provider string) error {
 	return nil
 }
 
-func (e *etcdMetricsMeasurement) getEtcdMetrics(host, provider string) ([]*model.Sample, error) {
+func (e *etcdMetricsMeasurement) getEtcdMetrics(host, provider string, port int) ([]*model.Sample, error) {
 	// Etcd is only exposed on localhost level. We are using ssh method
 	if provider == "gke" {
 		klog.Infof("%s: not grabbing etcd metrics through master SSH: unsupported for gke", e)
 		return nil, nil
 	}
 
 	// In https://github.com/kubernetes/kubernetes/pull/74690, mTLS is enabled for etcd server
-	// http://localhost:2382 is specified to bypass TLS credential requirement when checking
-	// etcd /metrics and /health.
-	if samples, err := e.sshEtcdMetrics("curl http://localhost:2382/metrics", host, provider); err == nil {
+	// in order to bypass TLS credential requirement when checking etc /metrics and /health, you
+	// need to provide the insecure http port number to access etcd, http://localhost:2381 for
+	// example.
+	cmd := "curl http://localhost:" + strconv.Itoa(port) + "/metrics"
```

Comment:
With port being int, I'd use sprintf to do the conversion for us: `cmd := fmt.Sprinf("curl http://localhost:%d/metrics", port)`

Reply:
done